src/app/views/panels/widgets/manager.tsx

```diff
@@ -152,6 +154,29 @@ export class WidgetManager implements Prototypes.Controls.WidgetManager {
     );
   }
 
+  public inputDate(
+    property: Prototypes.Controls.Property,
+    options: Prototypes.Controls.InputDateOptions = {}
+  ) {
+    const value = this.getPropertyValue(property) as number;
+    return (
+      <InputDate
+        {...options}
+        defaultValue={value}
+        onEnter={value => {
+          if (value == null) {
+            this.emitSetProperty(property, null);
+            return true;
+          } else if (value == value) {
```

Comment:
What does this condition do?  Are you trying to compare the `value` to the `value` above (on line 161)?

Reply:
It seems happened after merge and wrong resolving conflicts. Thanks!

Reply:
No, it's not issue of merging. I found several cases in other parts of code.

Reply:
Fixed